hipify_torch/hipify_python.py

```diff
@@ -600,11 +603,13 @@ def is_pytorch_file(rel_filepath):
         return True
     if rel_filepath.startswith("torch/"):
         return True
+    if rel_filepath.startswith("third_party/nvfuser/"):
+        return True
     if rel_filepath.startswith("tools/autograd/templates/"):
         return True
     return False
 
-def is_cusparse_file(rel_filepath):
+def is_special_file(rel_filepath):
     assert(not os.path.isabs(rel_filepath))
     if is_pytorch_file(rel_filepath):
         return "sparse" in rel_filepath.lower()
```

Comment:
You will also need to check for "linalg" in rel_filepath.lower() in order to pick up the hipsolver files.

Reply:
@alugorey yes this was intentionally not added as the support for hipsolver is not yet upstreamed which caused build failures in PyTorch. "linalg" will be added once the hipsolver PR is merged.